logger/logger.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Sonatype Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package logger has functions to obtain a logger, and helpers for setting up where the logger writes
+package logger
+
+import (
+	"os"
+	"path"
+
+	"github.com/sirupsen/logrus"
+)
+
+// DefaultLogFile can be overriden to use a different file name for upstream consumers
+var DefaultLogFile = "nancy.combined.log"
+
+// Logger can be obtained from outside the package
+var Logger *logrus.Logger
+
+func init() {
+	file, err := os.OpenFile(getLogFileLocation(), os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0644)
+	if err != nil {
+		Logger.Fatal(err)
+	}
+
+	logger := &logrus.Logger{
+		Out:          file,
+		Level:        logrus.DebugLevel,
```

Comment:
I see you used trace else where but there is no switch here. Are you thinking of adding more flags that can be passed to up the level. Maybe a good ole `-vvv` like in curl

Reply:
Yep! That's the idea. Maybe set the logger to Error by default, -v does Debug, -vv Info, -vvv Trace?

Reply:
-v == Info 
-vv == Debug
-vvv == Trace

Youve got Info and Debug flipped i think. But 👍

Reply:
Hah, likely. I never understood the levels anyways to begin with :)